scripts/gen_vimdoc.py

```diff
@@ -171,14 +230,13 @@
         'append_only': [],
     },
 }
+# fmt: on
 
-param_exclude = (
-    'channel_id',
-)
+param_exclude = ("channel_id",)
 
 # Annotations are displayed as line items after API function descriptions.
 annotation_map = {
-    'FUNC_API_FAST': '{fast}',
+    "FUNC_API_FAST": "{fast}",
```

Comment:
I thought our style or lint specified single quotes. At least that's what https://github.com/psf/black does IIRC, and that is probably how we should format our python code...

Single quotes are easier to type. Is there a reason to use double quotes?